### ISSUE TITLE
Question about positive definiteness of C

### ISSUE THREAD

**Issue by rohanb2018** [CMA-ES/pycma#203]
Hello, I had a general question about positive-definiteness of the `C` matrix when running `CMAEvolutionStrategy`. I noticed once when running the algorithm for my particular use-case that the covariance matrices seemed to become non-positive-definite, where the minimal eigenvalue became a very large negative value (on the order of `-1e306`), and then became `-np.inf` at a generation that followed.

I was wondering if there were any general insights as to when this type of situation can happen. Is there something about my fitness function that is ill-conditioned somehow, or should I investigate tuning the CMA hyper-parameters (e.g. population size)? I saw an issue that could be related (#96) and the dimension of my parameter space is around ~1000, so I guess there is a very small probability that the covariance matrix can be non-positive-definite in some runs?

Thanks!

**Comment by nikohansen** [comment]
Thanks for report! I would consider this to be a bug. Could you share the calling code (such that we can see the options used), and ideally the observed output, maybe the figure when `cma.plot()` is called?

A simple "fix" for the time being should be to set the `CMA_active` option to `False`.

**Comment by nikohansen** [comment]
Thanks, that helps already a lot, could you also provide the version of the code from `cma.__version__`, as it doesn't seem you are using the latest version if it.

**Comment by rohanb2018** [comment]
Yup sure! I have version 3.0.3 (output of `cma.__version__` is `3.0.3  $Revision: 4430 $ $Date: 2020-04-21 01:19:04 +0200 (Tue, 21 Apr 2020) $`)

Perhaps I should upgrade to the latest version of CMA and see whether the issue persists?

**Comment by nikohansen** [comment]
Thanks! Most importantly, I want to be able to replicate the issue or get more information on the state of the class instance before it happens (like via the plots) and thereby find the mechanism how this is happening. I don't mind which version I would need to use for the replication. I am also not very confident that the issue has already been fixed in the current version.
